### PR TITLE
Checklist: Extract shared component for video checks

### PR DESCRIPTION
**Pull request by swissspidy**
## Summary

Just making this code a bit more DRY as there was lots of repetition.

### QA

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [ ] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #

### CODE REVIEW

File: assets/src/edit-story/components/checklist/checks/imageElementResolution.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ const ImageElementResolution = () => {
     [setHighlights]
   );
 
-  const { footer, title } = DESIGN_COPY.lowImageResolution;
+  const { footer, title } = DESIGN_COPY.imageResolutionTooLow;
```
Comment (swissspidy) [inline assets/src/edit-story/components/checklist/checks/imageElementResolution.js:87]: Just to match `videoResolutionTooLow`

File: assets/src/edit-story/components/checklist/constants.js [hunk 2]
```diff
@@ -277,7 +277,7 @@ export const DESIGN_COPY = {
       `${MIN_VIDEO_RESOLUTION}p`
     ),
   },
-  videoResolutionTooHigh: {
```
Comment (swissspidy) [inline assets/src/edit-story/components/checklist/constants.js:280]: This was simply incorrect as the message was never about video _resolution_ but video _duration_.

**Review by maxyinger — APPROVED** [review]
Looks good. makes sense. Thanks for the cleanup!

**Review by BrittanyIRL — APPROVED** [review]
😻  thank you for the clean up!

**Pull request merged**